Source/WebCore/platform/mac/ThemeMac.mm

```diff
@@ -321,44 +281,21 @@ static LengthSize radioSize(const LengthSize& zoomedSize, float zoomFactor)
 
     return sizeFromNSControlSize(NSControlSizeSmall, zoomedSize, zoomFactor, radioSizes());
 }
-    
-static void configureToggleButton(NSCell* cell, StyleAppearance appearance, OptionSet<ControlStyle::State> states, const IntSize& zoomedSize, float zoomFactor)
-{
-    setControlSize(cell, appearance == StyleAppearance::Checkbox ? checkboxSizes() : radioSizes(), zoomedSize, zoomFactor);
-    updateStates(cell, states);
-}
-    
-static RetainPtr<NSButtonCell> createToggleButtonCell(StyleAppearance appearance)
+
+static NSButtonCell *sharedRadioCell(const IntSize& zoomedSize, float zoomFactor)
 {
-    RetainPtr<NSButtonCell> toggleButtonCell = adoptNS([[NSButtonCell alloc] init]);
-    
-    if (appearance == StyleAppearance::Checkbox) {
-        [toggleButtonCell setButtonType:NSButtonTypeSwitch];
-        [toggleButtonCell setAllowsMixedState:YES];
```

Comment:
It looks like `setAllowsMixedState` has been completely removed. But I think that's fine since it only has painting implications, which `ThemeMac` is not responsible for?

Reply:
That was my rationale for removing it, yeah.

Reply:
Oh, we can remove the cell completely. It just calls `controlSizeFromPixelSize()` like switch. Had not realized that the first time around. 🎉